tests/cpp/test_multidevice_host_ir.cpp

```diff
@@ -0,0 +1,190 @@
+// clang-format off
+/*
+* SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+* All rights reserved.
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+// clang-format on
+#include <gtest/gtest.h>
+
+#include <codegen.h>
+#include <device_lower/lower2device.h>
+#include <disjoint_set.h>
+#include <executor.h>
+#include <executor_params.h>
+#include <expr_evaluator.h>
+#include <fusion.h>
+#include <fusion_segmenter.h>
+#include <ir/all_nodes.h>
+#include <ir/graphviz.h>
+#include <ir/iostream.h>
+#include <ir/printer.h>
+#include <ir/utils.h>
+#include <iter_visitor.h>
+#include <kernel_cache.h>
+#include <kernel_ir.h>
+#include <mma_type.h>
+#include <ops/all_ops.h>
+#include <root_domain_map.h>
+#include <scheduler/all_schedulers.h>
+#include <scheduler/reduction_utils.h>
+#include <scheduler/utils.h>
+#include <tests/cpp/multidevice.h>
+#include <torch/csrc/jit/codegen/cuda/interface.h>
+#include <transform_replay.h>
+#include <transform_rfactor.h>
+
+#include <algorithm>
+#include <iostream>
+
+#include <host_ir/container.h>
+#include <host_ir/executor.h>
+
+namespace nvfuser {
+
+namespace hir {
+
+using MultiDeviceHostIrTestParams = std::tuple<bool, bool>;
+
+class MultiDeviceHostIrTest
+    : public MultiDeviceTest,
+      public testing::WithParamInterface<MultiDeviceHostIrTestParams> {};
+
+/*
```

Comment:
```suggestion
//
```
for style consistency across the code base.